### PR TITLE
Auto OCP-27564

### PR DESCRIPTION
**Pull request by chao007**
@liangxia @duanwei33 Plsease help reivew.

### CODE REVIEW

**Comment by chao007** [comment]
Adding test log https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/72683/console

File: features/storage/operator.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  
+  # @author chaoyang@redhat.com
+  # @case_id OCP-27564
+  @admin
+  Scenario: CSI snapshot controller operator is installed by default
+    When I run the :get admin command with:
+      | resource | pod                                        |
+      | n        | openshift-csi-snapshot-controller-operator |
+    Then the step should succeed
+        And the output should contain:
```
Comment (duanwei33) [inline features/storage/operator.feature]: pls adjust the format

**Comment by duanwei33** [comment]
@chao007 
When checking pod status in the namespace, is it possible to check all pods or specific pod should be "running" but not only contain "running", it means it will pass even there is "non-running" pod?

**Comment by chao007** [comment]
> @chao007
> When checking pod status in the namespace, is it possible to check all pods or specific pod should be "running" but not only contain "running", it means it will pass even there is "non-running" pod?

There should be only one pod.

**Comment by duanwei33** [comment]
Yes, I see it's only one pod, just remind it should be tracked if pod adding in the following release.

**Comment by chao007** [comment]
@pruan-rht Please help merge this

**Comment by chao007** [comment]
@liangxia Seems need your approval.

**Comment by liangxia** [comment]
I would like replace it with https://github.com/openshift/cucushift/pull/7413

**Comment by liangxia** [comment]
openshift/cucushift#7413 merged a week ago.

**Pull request closed without merging**